lib/ui/src/Cardano/Wallet/UI/Lib/Address.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE QuasiQuotes #-}
+module Cardano.Wallet.UI.Lib.Address
+where
+
+import Prelude
+
+import Cardano.Ledger.Address
+    ( unCompactAddr
+    )
```

Comment:
```suggestion
import Cardano.Wallet.Read.Address
    ( toShortByteString
    )
```

We have our own thing now. 🤓  The idea is that `cardano-wallet-read` provides all the functionality needed to work `Address`. If it does not, I'm happy to add it. (And I'm also happy to park functionality elsewhere, as long as we clean it up later.)